### PR TITLE
Adding Toolbar v1.0

### PR DESCRIPTION
**Pull request by ikostova-Tick42**

### CODE REVIEW

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community. We could not parse the GitHub identity of the following contributors: **ikostova**.
This is most likely caused by a git client misconfiguration; please make sure to:
1. check if your git client is configured with an email to sign commits `git config --list | grep email`
2. If not, set it up using `git config --global user.email email@example.com`
3. Make sure that the git commit email is configured in your GitHub account settings, see https://github.com/settings/emails

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community! We require contributors to sign a [Contributor License Agreement](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements#LegalRequirements-ContributorLicenseAgreement) and we don't seem to have CLAs on file for these contributors to the Pull Request: (@ibaltadzhieva). In order for your PR to be reviewed and merged, please follow the directions at the link above.

Project team: please **do not merge this Pull Request** until Foundation staff have confirmed that a CLA is in place for the new contributor(s) listed above.

If there are any questions, please don't hesitate to [get in touch with our Infrastructure Support team](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/77955298/Getting+in+Touch#GettinginTouch-InfrastructureSupport). /CC @finos-admin

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community. We could not parse the GitHub identity of the following contributors: **ikostova**.
This is most likely caused by a git client misconfiguration; please make sure to:
1. check if your git client is configured with an email to sign commits `git config --list | grep email`
2. If not, set it up using `git config --global user.email email@example.com`
3. Make sure that the git commit email is configured in your GitHub account settings, see https://github.com/settings/emails

File: toolbar-ui-angular/package.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    "@angular/compiler-cli": "~7.2.0",
+    "@angular/language-service": "~7.2.0",
+    "@types/electron": "^1.6.10",
+    "@types/jasmine": "~2.8.8",
+    "@types/jasminewd2": "~2.0.3",
+    "@types/node": "~8.9.4",
+    "codelyzer": "~4.5.0",
+    "electron": "^4.0.4",
+    "electron-packager": "^13.0.1",
+    "glue-interop-api-impl": "^1.1.4",
```
Comment (ghost) [inline toolbar-ui-angular/package.json:48]: This dependency was not found.

Comment (maoo) [reply toolbar-ui-angular/package.json:48]: When contributed, I've validated the build of the production bundle, which completes successfully, although this missing dependency gets flagged as optional but missing:

```
npm WARN @plexus-interop/transport-common@0.4.0 requires a peer of rxjs@^5.4.2 but none is installed. You must install peer dependencies yourself.
npm WARN @plexus-interop/websocket-transport@0.4.0 requires a peer of rxjs@^5.4.2 but none is installed. You must install peer dependencies yourself.
npm WARN finos-fdc3-appd-toolbar@1.0.0 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: tick42-glue-core@4.3.4 (node_modules/tick42-glue-core):
npm WARN 404 SKIPPING OPTIONAL DEPENDENCY: Not Found - GET https://registry.npmjs.org/tick42-glue-core - Not found
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: glue-interop-api-impl@1.1.4 (node_modules/glue-interop-api-impl):
npm WARN 404 SKIPPING OPTIONAL DEPENDENCY: Not Found - GET https://registry.npmjs.org/glue-interop-api-impl - Not found
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: com-glue42-finos-fdc3-api-impl@1.0.3 (node_modules/com-glue42-finos-fdc3-api-impl):
npm WARN 404 SKIPPING OPTIONAL DEPENDENCY: Not Found - GET https://registry.npmjs.org/com-glue42-finos-fdc3-api-impl - Not found
```

These the steps to reproduce it:
```
npm install -g node-license-validator
git clone git@github.com:Tick42/finos-fdc3-appd-toolbar.git
cd finos-fdc3-appd-toolbar/
npm install --production
cd toolbar-ui-angular
npm install --production
```

More info on [this JIRA comment](https://finosfoundation.atlassian.net/browse/CONTRIB-43?focusedCommentId=12485&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-12485); hope this helps.

**Comment by maoo** [comment]
@finos-cla-bot[bot] check

**Comment by finos-admin** [comment]
Thank you for your pull request and welcome to our community. We could not parse the GitHub identity of the following contributors: **ikostova**.
This is most likely caused by a git client misconfiguration; please make sure to:
1. check if your git client is configured with an email to sign commits `git config --list | grep email`
2. If not, set it up using `git config --global user.email email@example.com`
3. Make sure that the git commit email is configured in your GitHub account settings, see https://github.com/settings/emails

**Comment by finos-admin** [comment]
The cla-bot has been summoned, and re-checked this pull request!

**Comment by brooklynrob** [comment]
As the FDC3 PMC does its checks, please do note there are two sets of commits on this PR, including a set from April 24, at https://github.com/FDC3/appd-launcher-poc/pull/1/commits/348fbb0af6c84428cc924f04cf1d9cefd932afa0

File: toolbar-ui-angular/package.json (hunk 1 above)
Comment (brooklynrob) [reply toolbar-ui-angular/package.json:48]: @ikostova-Tick42 @lspiro-Tick42 Any background or context that the FDC3 PMC should consider re the change from "glue-fdc3-access-impl": "^1.0.11" to glue-interop-api-impl": "^1.1.4" does in the 2nd set of commits on this PR from 4/24?

**Comment by rikoe** [comment]
Closing in favour of #2

**Pull request closed without merging**